definitions/divorce/json/CaseEvent/CaseEvent-deemed-and-dispensed-nonprod.json

```diff
@@ -201,8 +201,8 @@
     "LiveFrom": "16/09/2020",
     "CaseTypeID": "DIVORCE",
     "ID": "coRespAnswerReceivedFromServiceAppRejected",
-    "Name": "Answer received",
-    "Description": "Answer received",
+    "Name": "Answer received - CoRespondent",
+    "Description": "Answer received - CoRespondent",
```

Comment:
Description should be: "Answer received from Co-Respondent"

Reply:
Please read the description of the PR. I also commented on the ticket

Reply:
Oh okay nice, yep that's okay 👌